src/components/Label/Label.types.ts

```diff
@@ -13,33 +13,42 @@ export enum LabelSize {
 
 export interface LabelIconButtonProps extends ButtonProps {
   /**
-   * The label icon button is shown
+   * The label icon button is shown.
    * @default false
    */
   show?: boolean;
   /**
-   * Content to show on the tooltip
+   * The tooltip props.
+   * @default false
+   */
+  tooltipprops?: TooltipProps;
```

Comment:
any reason not to camelcase as tooltipProps? reads a bit better with the consecutive p's

Reply:
Will update. React displays warnings using camelcase attributes, but that's okay.